Build errors on Windows 10

building SIRF `master` on my Windows 10 laptop produces lots of error messages like this one:
```
10>sirf_inverse_transform.obj : error LNK2019: unresolved external symbol "protected: virtual class sirf::NiftiImageData3DDeformation<float> * __cdecl sirf::NiftiImageData3DDeformation<float>::get_inverse_impl_nr(class std::shared_ptr<class sirf::NiftiImageData<float> const >)const " (?get_inverse_impl_nr@?$NiftiImageData3DDeformation@M@sirf@@MEBAPEAV12@V?$shared_ptr@$$CBV?$NiftiImageData@M@sirf@@@std@@@Z) referenced in function "public: class sirf::NonRigidTransformation<float> * __cdecl sirf::NiftiImageData3DDeformation<float>::get_inverse_impl_nr(class std::shared_ptr<class sirf::NiftiImageData<float> const >)const " (?get_inverse_impl_nr@?$NiftiImageData3DDeformation@M@sirf@@QEBAPEAV?$NonRigidTransformation@M@2@V?$shared_ptr@$$CBV?$NiftiImageData@M@sirf@@@std@@@Z)
10>sirf_inverse_transform.obj : error LNK2019: unresolved external symbol "protected: virtual class sirf::NiftiImageData3DDeformation<float> * __cdecl sirf::NiftiImageData3DDeformation<float>::get_inverse_impl_vtk(class std::shared_ptr<class sirf::NiftiImageData<float> const >)const " (?get_inverse_impl_vtk@?$NiftiImageData3DDeformation@M@sirf@@MEBAPEAV12@V?$shared_ptr@$$CBV?$NiftiImageData@M@sirf@@@std@@@Z) referenced in function "public: class sirf::NonRigidTransformation<float> * __cdecl sirf::NiftiImageData3DDeformation<float>::get_inverse_impl_vtk(class std::shared_ptr<class sirf::NiftiImageData<float> const >)const " (?get_inverse_impl_vtk@?$NiftiImageData3DDeformation@M@sirf@@QEBAPEAV?$NonRigidTransformation@M@2@V?$shared_ptr@$$CBV?$NiftiImageData@M@sirf@@@std@@@Z)
10>sirf_inverse_transform.obj : error LNK2019: unresolved external symbol "protected: virtual class sirf::NiftiImageData3DDisplacement<float> * __cdecl sirf::NiftiImageData3DDisplacement<float>::get_inverse_impl_nr(class std::shared_ptr<class sirf::NiftiImageData<float> const >)const " (?get_inverse_impl_nr@?$NiftiImageData3DDisplacement@M@sirf@@MEBAPEAV12@V?$shared_ptr@$$CBV?$NiftiImageData@M@sirf@@@std@@@Z) referenced in function "public: class sirf::NonRigidTransformation<float> * __cdecl sirf::NiftiImageData3DDisplacement<float>::get_inverse_impl_nr(class std::shared_ptr<class sirf::NiftiImageData<float> const >)const " (?get_inverse_impl_nr@?$NiftiImageData3DDisplacement@M@sirf@@QEBAPEAV?$NonRigidTransformation@M@2@V?$shared_ptr@$$CBV?$NiftiImageData@M@sirf@@@std@@@Z)
10>sirf_inverse_transform.obj : error LNK2019: unresolved external symbol "protected: virtual class sirf::NiftiImageData3DDisplacement<float> * __cdecl sirf::NiftiImageData3DDisplacement<float>::get_inverse_impl_vtk(class std::shared_ptr<class sirf::NiftiImageData<float> const >)const " (?get_inverse_impl_vtk@?$NiftiImageData3DDisplacement@M@sirf@@MEBAPEAV12@V?$shared_ptr@$$CBV?$NiftiImageData@M@sirf@@@std@@@Z) referenced in function "public: class sirf::NonRigidTransformation<float> * __cdecl sirf::NiftiImageData3DDisplacement<float>::get_inverse_impl_vtk(class std::shared_ptr<class sirf::NiftiImageData<float> const >)const " (?get_inverse_impl_vtk@?$NiftiImageData3DDisplacement@M@sirf@@QEBAPEAV?$NonRigidTransformation@M@2@V?$shared_ptr@$$CBV?$NiftiImageData@M@sirf@@@std@@@Z)
```

I have the opposite:
```
 -- One or more of the following components are missing: Registration, ISMRMRD and STIR (with ITK). Synergistic code will not be built.
11>  -- STIR not built with ITK.
11>  -- Either Registration or ISMRMRD or STIR (with ITK) have not been built.
...
C:\Users\krisf\Documents\devel\build-VC\SIRF-SuperBuild\sources\SIRF\src\Registration\cReg\NiftiImageData3DDeformation.cpp(118): error C2908: explicit specialization; 'sirf::NonRigidTransformation<dataType> *sirf::NiftiImageData3DDeformation<dataType>::get_inverse_impl_nr(const std::shared_ptr<const sirf::NiftiImageData<dataType>>) const' has already been instantiated [C:\Users\krisf\Documents\devel\build-VC\SIRF-SuperBuild\builds\SIRF\build\src\Registration\cReg\Reg.vcxproj]
...
C:\Users\krisf\Documents\devel\build-VC\SIRF-SuperBuild\sources\SIRF\src\Registration\cReg\NiftiImageData3DDeformation.cpp(153): error C2908: explicit specialization; 'sirf::NonRigidTransformation<dataType> *sirf::NiftiImageData3DDeformation<dataType>::get_inverse_impl_vtk(const std::shared_ptr<const sirf::NiftiImageData<dataType>>) const' has already been instantiated [C:\Users\krisf\Documents\devel\build-VC\SIRF-SuperBuild\builds\SIRF\build\src\Registration\cReg\Reg.vcxproj]
```
and same errors for `NiftiImageData3DDisplacement.cpp`.

This is with `USE_ITK=OFF` in the SuperBuild. Checking the SIRF CMakeCache.txt it has
```
VTK_DIR=VTK_DIR-NOTFOUND
```

--- a/src/Registration/cReg/include/sirf/Reg/NonRigidTransformation.h
+++ b/src/Registration/cReg/include/sirf/Reg/NonRigidTransformation.h
@@ -46,6 +46,8 @@ public:
     /// Destructor
     virtual ~NonRigidTransformation() {}
 
+#ifndef _MSC_VER // need to disable on Visual Studio for https://github.com/SyneRBI/SIRF/issues/665
+
     /*! \brief Get inverse as unique pointer (potentially based on another image).
      *
      * Why would you want to base it on another image? Well, we might have a deformation
@@ -67,5 +69,7 @@ protected:
 
     /// Helper function for get_inverse (VTK). Don't use.
     virtual NonRigidTransformation* get_inverse_impl_vtk(const std::shared_ptr<const NiftiImageData<dataType> > image_sptr = nullptr) const = 0;
+#endif // _MSC_VER
 };
+
 }

